src/AutorestSwift/View Models/ConstantViewModel.swift

```diff
@@ -35,7 +35,7 @@ struct ConstantViewModel {
     let comment: ViewModelComment
     // default value of the proeprty
     let defaultValue: ViewModelDefault
-    let serializedName: String
+    let serializedName: String?
```

Comment:
From the way how we set ` serializedName`, I think serializedName will not never nil.

Reply:
I think this change is not needed.

Reply:
Reverted.